pkg/connector/handlematrix.go

```diff
@@ -676,3 +679,70 @@ func (t *MetaClient) HandleMatrixDeleteChat(ctx context.Context, chat *bridgev2.
 	}
 	return fmt.Errorf("unknown platform for deleting chat: %v", platform)
 }
+
+func (m *MetaClient) HandleMatrixRoomName(ctx context.Context, msg *bridgev2.MatrixRoomName) (bool, error) {
+	if msg.Portal.RoomType == database.RoomTypeDM {
+		return false, fmt.Errorf("renaming not supported in DMs")
+	}
+	platform := m.LoginMeta.Platform
+	threadID := metaid.ParseFBPortalID(msg.Portal.ID)
+	if platform == types.Instagram {
+		err := m.Client.Instagram.EditGroupTitle(ctx, strconv.FormatInt(threadID, 10), msg.Content.Name)
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	} else if platform == types.Facebook || platform == types.Messenger {
+		_, err := m.Client.ExecuteTasks(ctx, &socket.RenameThreadTask{
+			ThreadKey:  threadID,
+			ThreadName: msg.Content.Name,
+			SyncGroup:  1,
+		})
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	}
+	return false, fmt.Errorf("unknown platform for renaming chat: %v", platform)
+}
+
+func (m *MetaClient) HandleMatrixRoomAvatar(ctx context.Context, msg *bridgev2.MatrixRoomAvatar) (bool, error) {
+	if msg.Portal.RoomType == database.RoomTypeDM {
+		return false, fmt.Errorf("changing avatar not supported in DMs")
+	}
+	if m.LoginMeta.Platform == types.Instagram {
+		// TODO: implement Instagram avatar changing. IG Web doesn't support this.
+		return false, fmt.Errorf("changing avatar not supported on Instagram")
+	}
+	threadID := metaid.ParseFBPortalID(msg.Portal.ID)
+	var imageID int64
+	if msg.Content.URL == "" && msg.Content.MSC3414File == nil {
+		// TODO: handle removing avatar. Messenger web doesn't have a remove option?
```

Comment:
Same as above

Reply:
Did you try just setting a blank/zero image ID?

Reply:
Yes